### PR TITLE
Release + added new release script

### PR DESCRIPTION
**Pull request by j-mracek**

### CODE REVIEW

**Comment by j-mracek** [comment]
test this please

File: .tito/packages/dnf-plugins-extras [hunk 1]
```diff
@@ -0,0 +1 @@
+1.0.0-1 ./
```
Comment (ignatenkobrain) [inline .tito/packages/dnf-plugins-extras]: No shito please

File: dnf-plugins-extras.spec [hunk 2]
```diff
@@ -523,3 +523,19 @@ PYTHONPATH="%{buildroot}%{python3_sitelib}:%{buildroot}%{python3_sitelib}/dnf-pl
 %{python3_sitelib}/dnf-plugins/__pycache__/torproxy.*
 
 %changelog
+* Fri Feb 10 2017 Jaroslav Mracek <jmracek@redhat.com> 1.0.0-1
```
Comment (ignatenkobrain) [inline dnf-plugins-extras.spec]: No changeling in upstream, only in downstreams

File: scripts/update_releasenotes.py [hunk 3]
```diff
@@ -0,0 +1,1636 @@
+#!/usr/bin/python2
```
Comment (ignatenkobrain) [inline scripts/update_releasenotes.py]: What's the point of copy-pasting same script over each repos?

File: dnf-plugins-extras.spec (hunk 2 above)
Comment (ignatenkobrain) [inline dnf-plugins-extras.spec]: Changelog*

File: scripts/update_releasenotes.py (hunk 3 above)
Comment (j-mracek) [inline scripts/update_releasenotes.py]: The reason is easy, same path, same command, same changelog like in dnf

File: .tito/packages/dnf-plugins-extras (hunk 1 above)
Comment (j-mracek) [inline .tito/packages/dnf-plugins-extras]: No idea what are you talking about, probably I have to keep tito commit

**Pull request closed without merging**